main.js

```diff
@@ -13,52 +13,71 @@ const hoursWeekSleep = document.getElementById('hours-week-sleep');
 const qualityWeekSleep = document.getElementById('quality-week-sleep');
 const allTimeAvgSleep = document.getElementById('alltime-avg-sleep');
 const allTimeAvgHydration = document.getElementById('alltime-avg-hydration');
-const stepsToday = document.getElementById('today-steps')
-const activityToday = document.getElementById('today-activity')
+const stepsToday = document.getElementById('today-steps');
+const activityToday = document.getElementById('today-activity');
+const friendTrends = document.getElementById('friend-trends');
+const topSteps = document.getElementById('highest-steps');
 
-const loadUser = () => {
```

Comment:
Good thinking taking everything out of the function and just having it all just happen on page load! I might look into document.ready and put some of these things in that method to make sure the DOM is fully loaded before we populate it.